Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -13,11 +13,11 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 0,
-        "Patch": 6
+        "Minor": 1,
+        "Patch": 0
     },
     "demands": [],
-    "minimumAgentVersion": "2.0.0",
+    "minimumAgentVersion": "2.1.0",
```

Comment:
Ok, I found some documentation and I now understand. I have reverted the change.